### ISSUE TITLE
Teach keyboard shortcuts to new players

### ISSUE THREAD

**Issue by hbnrmx** [ppy/osu#5449]
**Describe the new feature:** 

I was looking through the old client and discovered a lot of cool keyboard shortcuts
(e.g. `F1` -> Open Mod overlay, `1` -> Deselect All, `2` -> Close)

Most (if not all) of them have been carried over to Lazer. 
However, there's currently no way to discover them.

Possible ways to teach new players about keyboard shortcuts:

1. Hold down `ALT` to change the button's texts (similar to Windows 10 File Explorer)
1. add back Tips to the Home screen (could prove to be useful for other things as well)
1. some combination of 1 + 2 (e.g. "Tip: Hold down `ALT` to discover useful keyboard shortcuts!")
1. Tooltips (easier to implement, especially with buttons that don't have text like MUTE)
1. ...

**Comment by peppy** [comment]
Yep, we plan on having a method of discovering keyboard shortcuts. I discussed with @arflyte but I don't believe he has come up with a design yet.

**Comment by memesnightmares** [comment]
This appears to have been largely resolved in #791. However, there are still some undocumented shortcuts such as CTRL+Enter to autoplay a map (and possibly others that I haven't discovered yet).
